dev-docs/lit-labs-lifecycle.md

```diff
@@ -0,0 +1,196 @@
+
+The impetus for a formal "labs" designation stemmed from a desire to balance a tension between (a) the need to publish exploratory code to npm in order to get real-world feedback on new product ideas, and (b) setting expectations around quality/support for those packages, given that user feedback may drive significant changes or cause reconsideration for the product conception altogether.
+
+However, the aim is that once a labs package has reached a certain level of maturity, it will shed its "labs" status and graduate to a fully supported product (moving out of the labs folder in the monorepo and being published to the @lit npm org). This proposal suggests guidelines for advancing a labs package through a series of phases from exploration to production.
+
+## Proposal
+
+This proposal primarily seeks to ensure we have reasonably positive answers to the following questions before graduating a package from labs to "production".
+
+1. Have enough users tried the package and given positive feedback?
+2. Are all major known bugs resolved?
+3. Is the API considered reasonably stable?
+4. Do we have a plan to dedicate sufficient resources to support production users of the package?
+
+To this end, a labs package will go through three phases:
+
+- **Development**: Code is starting to be checked in, but is not yet ready for consumption/feedback. We may publish pre-releases to npm where useful for development, but code won't otherwise be promoted or obviously discoverable.
+
+- **Evaluation**: Releases are published to npm under @lit-labs for evaluation purposes. README has sufficient documentation to explain usage suitable for evaluation. The labs page on lit.dev describes the package and solicits for feedback from a specific discussion thread. Feedback via discussions & issues may drive frequent breaking changes.
+
+- **Production**: Release considered stable, published to npm under @lit org, and has dedicated docs page(s) and API documentation on lit.dev. Bugs and feature requests will be triaged and prioritized at the same level as other core packages. Breaking changes may happen, should be minimized to the extent possible.
+
+### Phase summary
+
+<table>
+  <tr>
+    <th>Phase:</th>
+    <th>Development</th>
+    <th>Evaluation</th>
+    <th>Production</th>
+  </tr>
+  <tr>
+    <th>
+      External usage expectation
+    </th>
+    <td>
+      None. Code is under development and not suitable for 3rd-party usage.
+    </td>
+    <td>
+      Code is published but recommended for evaluation purposes only.
+    </td>
+    <td>
+      Code is recommended for use in production.
+    </td>
+  </tr>
+  <tr>
+    <th>
+      Source location
+    </th>
+    <td colspan="2">
+      `packages/labs/{name}`
```

Comment:
I believe for markdown to apply within an html block, there must be an empty newline either side of this line.

E.g.

```md
<td colspan="2">

  `packages/labs/{name}`

</td>
```

Using GitHub comment markdown as a live preview, this results in:

<td colspan="2">

  `packages/labs/{name}`

</td>

vs 

<td colspan="2">
  `packages/labs/{name}`
</td>